.github/workflows/publish-build.yml

```diff
@@ -0,0 +1,60 @@
+# Copyright 2023 Practically.io All rights reserved
+
+name: Publish Build
+
+on:
+  push: {branches: ['1.x']}
+  workflow_run:
+    workflows:
+      - 'CI'
+    types:
+      - 'completed'
+
+jobs:
+  build:
+    name: Build Application
+    runs-on: ubuntu-latest
+    outputs:
+      build_output: ${{ steps.build_output.outputs.}}
```

Comment:
Do you think it's worth now using multi container builds rather than trying to pass around artefacts? This seemed to work a lot better on GitLab.

Reply:
Yeah I'm happy for a more concise/streamlined implementation. At the time this was the only way I could figure out how to do it